pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -46,7 +46,8 @@ def get_stationfile_name(station_name, obs_name, var_name_web, vert_code):
 
 def get_json_mapname(obs_name, var_name_web, model_name, model_var, vert_code, period):
     """Get name base name of json file"""
-    return f"{obs_name}-{var_name_web}_{vert_code}_{model_name}-{model_var}_{period}.json"
+    periodmod = period.replace("/", "")
```

Comment:
I am actually not sure why this was introduced, we could also have had filenames written with slashes for cams2-83.. anyway they are expected without now on aeroval so I'll write a comment

Reply:
I am not sure it's even relevant anymore now that we have switched to `_get_min_max_year_periods`, I will test and get rid of it if so

Reply:
actually I am not sure about what's going on here because if I remove the replace, the code crashes with FileNotFoundError, like 
```
FileNotFoundError: [Errno 2] No such file or directory: 
'/lustre/storeB/users/charlien/cams2-83-tests/data/cams2-83/test-forecast-day/ma
p/EEA-NRT-concno2_Surface_ENSEMBLE-concno2_2024/03/01.json'
```
but I was assuming this is just where the output filename is established since in coldatatojson_engine.py we have
```
            map_name = get_json_mapname(
                obs_name, var_name_web, model_name, model_var, vert_code, period
            )
            outfile_map = os.path.join(out_dirs["map"], map_name)
            write_json(map_data, outfile_map, ignore_nan=True)
``` 
but apparently not? I am not sure I follow why it crashes without the replace..
also I do not know when exactly do the periods as strings acquire slashes, they are not written with slashes in the config, must be some to-string conversion in ColocatedData object....

Reply:
I get it, the slash comes from aeroval/helpers.py
```
        if years:
            _per = "-".join([str(int(val)) for val in spl])
        else:
            _per = "-".join([to_pandas_timestamp(val).strftime("%Y/%m/%d") for val in spl])
```
I suspect that if I get rid of it here I can skip the periodmod thing, I will test

Reply:
seems to be the case